src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1050,15 +1055,36 @@ GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(std::string_vie
             return action;
         }
 
+        std::vector<DukValue> pluginCallArgs;
+        if (GetTargetAPIVersion() <= API_VERSION_68_CUSTOM_ACTION_ARGS)
+        {
+            pluginCallArgs = { *dukArgs };
+        }
+        else
+        {
+            log_info(
+                "Current player id = %i, action player id = %i", network_get_current_player_id(), customAction->GetPlayer());
```

Comment:
Please remove these, or use `log_verbose`, otherwise it polutes stdout which isn't great when developing plugins.

Reply:
Ah yes these were from testing and not meant to be committed. They are removed now. :)